feat: added choices for dealing with hungry looking marshmallows

gmenghini opened the pull request:
Please review this, @casey-collab

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/udacity/create-your-own-adventure/1923)
<!-- Reviewable:end -->

casey-collab commented:
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

casey-collab commented:
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

gmenghini commented:
I'm not quite sure what to make of this. When I try to update my local master branch by pulling from GitHub, there are no changes. Terminal output copied below:
garrettmenghini (gmenghini-edits) jump-rope $ git checkout master
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
garrettmenghini (master) jump-rope $ git pull origin master
From https://github.com/gmenghini/create-your-own-adventure
 * branch            master     -> FETCH_HEAD
Already up-to-date.
garrettmenghini (master) jump-rope $ git checkout gmenghini-edits
Switched to branch 'gmenghini-edits'
garrettmenghini (gmenghini-edits) jump-rope $ git merge master
Already up-to-date.
